src/app/icd/client/BUILD.gn

```diff
@@ -41,6 +41,8 @@ source_set("handler") {
     "CheckInHandler.h",
     "DefaultCheckInDelegate.cpp",
     "DefaultCheckInDelegate.h",
+    "ICDRefreshKeyInfo.cpp",
```

Comment:
ICDRefreshKeyInfo is used to register new key, and this file has been put inside ICD folder, maybe rename it to RefreshKeySender?

Reply:
Done.